.github/docker/crossbundle.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM androidsdk/android-31
+FROM androidsdk/android-30
```

Comment:
What is the reason for migrating from version 31?

Reply:
`Build crossbundle example from docker` failed so I tried to change the android version. I guess it could fix the strange error `detected conflict in cargo.lock file`. Maybe we should test other android versions like 32 and 33.